xcodecommand/mock_xcprettyManager.go

```diff
@@ -0,0 +1,90 @@
+// Code generated by mockery v2.13.1. DO NOT EDIT.
```

Comment:
Shouldn't this be in the root mock dir too?

Reply:
It is mostly used in the `xcpretty_test.go`, there is no need to mock it outside from this package, as the mock interface is used internally. There was preexisting xcpretty installation code moved and adapted in this pacakge, that is why it is exists.